pages/sponsorship.js

```diff
@@ -0,0 +1,168 @@
+import Card, { CardHeader, CardTitle, CardContent, CardButtonContainer } from '../components/card'
+import styled from 'styled-components'
+import Button from '../components/button'
+import React, { useState } from 'react'
+import Modal from 'react-modal'
+import { EDIT, NEW } from '../constants'
+
+
+const customStyles = {
+    content : {
+      top                   : '50%',
+      left                  : '50%',
+      right                 : 'auto',
+      bottom                : 'auto',
+      marginRight           : '-50%',
+      transform             : 'translate(-50%, -50%)'
+    }
+  };
+
+
+const TableDiv = styled.div`
+    border: 1px solid #000000;
+    border-radius: 3px;
+    padding: 1px 1px;
+    background-color: #FFFFFF;
+`
+
+const HeaderCard = styled.div`
+    padding: 26px 28px;
+    font-weight: bold;
+    font-family: Apercu Pro;
```

Comment:
We can probably set the font family higher up so we don't have to redefine it for every component